### ISSUE TITLE
Cannot find module 'ember-cli/lib/utilities/find-build-file'

### ISSUE THREAD

**Issue by doczoidberg** [angular/angular-cli#425]
I installed latest cli via npm and try to generate a new project `ng new new1` (windows). I get the following error:
```
Error: Cannot find module 'ember-cli/lib/utilities/find-build-file'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (c:\temp\node_modules\angular-cli\lib\cli\index.js:48:31)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
```

**Comment by hansl** [comment]
What's your `ng version` ?

**Comment by doczoidberg** [comment]
angular-cli: 0.0.31
node: 4.4.2
os: win32 x64

**Comment by doczoidberg** [comment]
local installation of angular-cli works. It has to do with my global installation which does not work. When reinstalling angular-cli as admin globally I get:

> npm ERR! Windows_NT 10.0.10240
> npm ERR! argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "angular-cli"
> npm ERR! node v4.4.2
> npm ERR! npm  v2.15.0
> npm ERR! path C:\Users\danie\AppData\Roaming\npm\node_modules\angular-cli\node_modules\ember-cli\node_modules\http-proxy\node_modules\requires-port\package.json.2796384356
> npm ERR! code EPERM
> npm ERR! errno -4048
> npm ERR! syscall rename
> 
> npm ERR! Error: EPERM: operation not permitted, rename 'C:\Users\danie\AppData\Roaming\npm\node_modules\angular-cli\node_modules\ember-cli\node_modules\http-proxy\node_modules\requires-port\package.json.2796384356' -> 'C:\Users\danie\AppData\Roaming\npm\node_modules\angular-cli\node_modules\ember-cli\node_modules\http-proxy\node_modules\requires-port\package.json'
> npm ERR!     at Error (native)
> npm ERR!  { [Error: EPERM: operation not permitted, rename 'C:\Users\danie\AppData\Roaming\npm\node_modules\angular-cli\node_modules\ember-cli\node_modules\http-proxy\node_modules\requires-port\package.json.2796384356' -> 'C:\Users\danie\AppData\Roaming\npm\node_modules\angular-cli\node_modules\ember-cli\node_modules\http-proxy\node_modules\requires-port\package.json']
> npm ERR!   errno: -4048,
> npm ERR!   code: 'EPERM',
> npm ERR!   syscall: 'rename',
> npm ERR!   path: 'C:\\Users\\danie\\AppData\\Roaming\\npm\\node_modules\\angular-cli\\node_modules\\ember-cli\\node_modules\\http-proxy\\node_modules\\requires-port\\package.json.2796384356',
> npm ERR!   dest: 'C:\\Users\\danie\\AppData\\Roaming\\npm\\node_modules\\angular-cli\\node_modules\\ember-cli\\node_modules\\http-proxy\\node_modules\\requires-port\\package.json' }
> npm ERR!
> npm ERR! Please try running this command again as root/Administrator.

**Comment by PascalPrecht** [comment]
Yea I think one interesting thing also to document (because I ran into this recently as well), is that a project generated using angular-cli has angular-cli as dependency and uses the local installation even tho, we think we use the global `ng` command. 

That was kind of surprising to me.

**Comment by doczoidberg** [comment]
I reinstalled node. Seems now to work for me.

**Comment by welkang** [comment]
so many issues, so shame...

**Comment by rahil471** [comment]
How is reinstalling node a solution?

**Comment by hansl** [comment]
This is from April 15. @rahil471 please explain why this 3 months old issue should be reopened.

**Comment by rahil471** [comment]
Never said it should be reopened @hansl 
It's just that, reinstalling node isn't the right solution to the problem. Need to get to the root why it is coming up.

**Comment by zfranklyn** [comment]
Installing hidden peer dependencies was my solution. I came across a similar issue, where the "missing module" sort of came out of nowhere, with no informative errors. My problem was I had just added a module that had peer dependencies, and NPM on Windows wasn't giving me missing peer dependency errors. I used the package `npm-install-peers` to help me install all my peers (and recorded them in package.json).
